Source/Core/DolphinQt/Debugger/MemoryViewWidget.h

```diff
@@ -18,7 +18,9 @@ class MemoryViewWidget : public QTableWidget
     U16,
     U32,
     ASCII,
-    Float32
+    ASCIIb,
```

Comment:
These shouldn't just be using `b` as a differentiator here. That isn't self-documenting about its meaning to anyone who isn't already acquainted with the code.